src/charm.py

```diff
@@ -236,6 +243,23 @@ def _on_pebble_ready(self, _):
         # proceed with other actions
         self._on_event(_)
 
+    def _send_manifests(self, interfaces, context, folder, relation):
+        """Send manifests from folder to desired relation."""
+        if relation in interfaces and interfaces[relation]:
+            manifests = self._create_manifests(folder, context)
+            interfaces[relation].send_data({relation: manifests})
+
+    def _create_manifests(self, folder, context):
+        """Create manifests string for given folder and context."""
+        manifests = []
+        manifest_files = glob.glob(f"{folder}/*.j2")
```

Comment:
I slightly favor writing these files out explicitly like [here](https://github.com/canonical/seldon-core-operator/blob/c34309f57403cd2a53d1216e816d52b083202b22/src/charm.py#L30) rather than slurping up the entire folder, mainly because I don't know if I'd expect adding a new file to that folder will immediately send it through this relation.  But if you prefer this way then lgtm

Reply:
explicit better than implicit. I am +1 on names list